python-cbtaxii.spec

```diff
@@ -1,15 +1,16 @@
 %define name python-cbtaxii
 %define version 1.6
 %define unmangled_version 1.6
-%define release 5
+%define release 7
```

Comment:
This is an incorrect use of the release portion of the version.  Since we are releasing a new version of this connector, we need to put its versioning in line with the rest of our connectors and linux standard in general.

Reply:
The rest of the connectors (at least the python ones) also use this variable to represent the last digit of a version number - is this really important enough to hold up release? Where in the connector code base do we use some other convention? 

`%define release 2

Name: python-cb-yara-connector
Version: %{version}
Release: %{release}%{?dist}
Summary: Carbon Black Yara Agent`

and so for threat connect